storagemarket/impl/client_environments.go

```diff
@@ -31,6 +32,10 @@ func (c *clientDealEnvironment) Node() storagemarket.StorageClientNode {
 	return c.c.node
 }
 
+func (c *clientDealEnvironment) CleanBlockStore(proposalCid cid.Cid) error {
```

Comment:
nit: let's use the same capitalization of Blockstore everywhere, eg CleanBlockstore